test/zipkin-simple.test.js

```diff
@@ -134,7 +134,7 @@ describe("Zipkin client", function () {
 				traceId: "test traceId",
 				spanId: "test spanId",
 				parentSpanId: "test parent_id",
-				timestamp: "test timestamp",
+				timestamp: new Date().getTime(),
```

Comment:
you're right, this is in milliseconds. 
This doesn't actually effect the test because we are using it only for the duration, but I'll add * 1000 for clarity